Cargo.toml

```diff
@@ -17,7 +17,7 @@ tonic-conn = ["tonic"]
 bytes = "1.3.0"
 futures = "0.3"
 libc = "0.2.138"
-vsock = "0.5.0"
+vsock = { git = "https://github.com/rust-vsock/vsock-rs", rev = "2223f5a" }
```

Comment:
I've published 0.5.1 with your changes now so you should be able to use that here.